### ISSUE TITLE
cURL request has failed: Operation timed out after 5001 milliseconds with 0 bytes received

### ISSUE THREAD

**Issue by Alderander** [BuycraftPlugin/BuycraftPM#23]
PocketMine-MP version 1.7dev-633

`[20:44:57] [Asynchronous Worker #1 thread/CRITICAL]: Exception: "cURL request has failed: Operation timed out after 5001 milliseconds with 0 bytes received" (EXCEPTION) in "BuycraftPM/src/Buycraft/PocketMine/PluginApi" at line 38
[20:44:57] [Asynchronous Worker #1 thread/DEBUG]: #0 BuycraftPM/src/Buycraft/PocketMine/Execution/DuePlayerCheck(45): Buycraft\PocketMine\PluginApi->basicGet(string /queue?limit=250&page=1)
[20:44:57] [Asynchronous Worker #1 thread/DEBUG]: #1 src/pocketmine/scheduler/AsyncTask(67): Buycraft\PocketMine\Execution\DuePlayerCheck->onRun()
[20:44:57] [Asynchronous Worker #1 thread/DEBUG]: #2 (): pocketmine\scheduler\AsyncTask->run()
[21:44:57] [Server thread/CRITICAL]: Could not execute asynchronous task DuePlayerCheck: Task crashed`

**Comment by dschwartz783** [comment]
Yes, I get this same thing at times. Sometimes it happens when the server starts up, but usually it's only after the server has been up for awhile. This exception should be handled in a try block to avoid this.
